CHANGELOG.md

```diff
@@ -13,11 +17,25 @@ _2017_
 -->
 
 # v0.1.0
-_2018-03-13_
-  - [Downloads for v0.1.0](https://github.com/okkur/dailymile/releases/tag/v0.1.0)
+_2017-01-01_
+  - [Downloads for v0.1.0](https://github.com/okkur/gomiler/releases/tag/v0.1.0)
   - [Changelog since v0.0.0](#changes-since-v000)
+  - [Documentation for v0.1.0](#documentation-for-v010)
+
+## Downloads for v0.1.0
```

Comment:
We link to the specific release page on Github so this subsection for Downloads is not necessary.